src/classes/STG_PanelRelAuto_CTRL.cls

```diff
@@ -133,19 +133,14 @@ public with sharing class STG_PanelRelAuto_CTRL extends STG_Panel {
 	            }   
 	            for (String s : fieldTokenMap.keySet()) {
 	                if (s.equalsIgnoreCase('ReportsToId') || (s.endsWith('__c') && !s.startsWith('npe01__') && !s.startsWith('npo02__') && !s.startsWith('npe03__'))){
-	                    //if we've hit the limit, throw a warning and recommend manual entry
-	                    if (system.Limits.getFieldsDescribes() == system.Limits.getLimitFieldsDescribes()) {
-	                        ApexPages.addMessage(new ApexPages.message(ApexPages.severity.WARNING, system.Label.npe4.RelationshipsCustomFieldOverload));                 
-	                    } else {
-	                        Schema.DescribeFieldResult F = fieldTokenMap.get(s).getDescribe();             
-	                        List <Schema.sObjectType> sobjref = new List<Schema.sObjectType>(); 
-	                        sobjref = F.getReferenceTo();
-	                        for (Schema.sObjectType objT : sobjref) {
-	                            if (objT == Contact.sObjectType) {
-	                                listSOACfield.add(new SelectOption(F.getName(), F.getLabel()));
-	                            }                   
-	                        }               
-	                    }           
+	                    Schema.DescribeFieldResult F = fieldTokenMap.get(s).getDescribe();             
```

Comment:
@njjc why did the describe check get removed?  i'm not saying we need it, but curious why it got yanked.